index.html

```diff
@@ -10,6 +11,66 @@
     <link rel="stylesheet" href="styles/index.css" />
 </head>
 <body>
+    <header>
+        <h1 id="weather-header" class="highlight-text">Weather Report</h1>
+    </header>
+
+    <main>
+        <section id="greeting">
+            <h2 id="greeting-message">Today’s skies over the beautiful city of <span id="current-city">Seattle</span>!</h2>
+        </section>
+
+        <section id="temperature" class="container">
+            <h3>Temperature</h3>
+            <div class= "center-align">
+                <button id="decrease-temp">-</button>
+                <span id="temp-number" class="big-text"></span>°F
+                <button id="increase-temp">+</button>
+                <button id="realtime-temperature" class="big-text">Realtime Temperature</button>
+            </div>
+        </section>
+
+        <section id="sky" class="container">
+            <h3>Sky</h3>
+            <label for="sky-select" class="big-text">Choose a sky:</label>
+            <select name="skys" id="sky-select" class="big-text">
+            <option value="">--Please choose an option--</option>
+            <option value="Sunny">Sunny</option>
+            <option value="Cloudy">Cloudy</option>
+            <option value="Rainy">Rainy</option>
+            <option value="Snowy">Snowy</option>
+            </select>
+            <div id="sky-emoji" class="big-text">☀️ ☀️ ☀️</div>
+            <img id="sky-img" class="uniform-image" src="assets/sunny.jpeg" alt="sky pic" ></img>
```

Comment:
You can remove the `</img>` here, since `<img>` is a self closing tag. In most browsers you wouldn't run into an error and it is tolerated. However, there are some that would raise an error. @kineticbelle @imlyj5 
```suggestion
            <img id="sky-img" class="uniform-image" src="assets/sunny.jpeg" alt="sky pic" >
```